src/Globalstyles.js

```diff
@@ -10,9 +10,8 @@ body {
   font-size: 16px;
   margin: 0;
   font-family: 'Roboto', sans-serif;
-  width:100vw;
-  height:100vh;
-  background:#E8F1F2; 
-  position:fixed;
+  width:100%;
+  height:100%;
+  background:#E8F1F2;  
```

Comment:
You use this background-color also in CardContainer.js. Maybe you can make a variable out of it?

Reply:
Yes, i have to think though if I need the CardContainer. I don't think I will need it anymore. I have to see how things develop.